src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -4007,12 +4007,12 @@ def cli(self, vrf_type='all', neighbor='', vrf='all', instance='all', address_fa
                     m2 = re.compile(r'^(?P<value>[0-9]+)'
                                  '(?P<space>\s{2,20})'
                                  '(?P<weight>[0-9]+) '
-                                 '(?P<path>[0-9\.\{\}\s]+)$').match(group_num)
+                                 '(?P<path>[()0-9\.\{\}\s]+)$').match(group_num)
     
                     # weight path
                     # 0 200 33299 51178 47751 {27016} 65000.65000
                     m3 = re.compile(r'^(?P<weight>[0-9]+) '
-                                 '(?P<path>(([\d\.]+\s)|(\{[\d\.]+\}\s))+)$')\
+                                 '(?P<path>(([()\d\.]+\s)|(\{[\d\.]+\}\s))+)$')\
```

Comment:
Please add examples of device output that your changes match above the patterns that you're modifying

Reply:
Done in [1ebacb6](https://github.com/CiscoTestAutomation/genieparser/pull/620/commits/1ebacb67a223d00c6bc9cc864130c31d801d3027)